Makefile

```diff
@@ -65,6 +66,7 @@ setup: force-upgrade
 	pre-commit install
 	pre-commit install --hook-type commit-msg
 	pre-commit install --hook-type pre-push
+	mkdir -p dist
```

Comment:
Should we add an explanation here, that the `build.yaml` action expects this folder to exist. I think a std reference link would be https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives?

Reply:
> Should we add an explanation here, that the `build.yaml` action expects this folder to exist. I think a std reference link would be https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives?

Done [here](https://github.com/jenstroeger/python-package-template/compare/3180dcab4f114ec74bf96124e00c0b60a8b12898..b0bae805034b104645bb7d2125574e71cf2877b6).